lochness/utils/path_checker.py

```diff
@@ -183,21 +204,41 @@ def update_interviews_teams_data_check(df: pd.DataFrame) -> pd.DataFrame:
 
     # file name pattern check
     video_int_df['file_pattern_check'] = video_int_df['file_name'].str.match(
-                r'\d{4}\d{2}\d{2}\d{6}_[A-Z]{2}\d{5}_(OPEN|PSYSCS).(wav|WAV)')
+        r'\d{4}\d{2}\d{2}\d{6}(wav|WAV)'
```

Comment:
No more required to include AMPSCZ-ID, "OPEN" or "PSYCHS"?

Reply:
Yep, we made these changes on Lochness' `config.yml` But did not update them here.

This new template in in-line with the Speech Team's SOP.